api/src/geolocate/get.handler.ts

```diff
@@ -0,0 +1,95 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+import * as AWS from 'aws-sdk';
+import { RDSDataService } from 'aws-sdk';
+import {
+  ExecuteStatementRequest,
+  FieldList,
+  SqlParametersList,
+} from 'aws-sdk/clients/rdsdataservice';
+
+const SCHEMA = 'public';
+
+/**
+ * Returns all geo identifiers that intersect with a lat, long.
+ * @param rdsService: RDS service to connect to the db.
+ * @param params: with lat and long coordinates sued to query tables.
+ * @param geoid: identifier to retrieve for a table.
+ * @param table: db table to query.
+ */
+async function getGeoDataForLatLong(
+  rdsService: RDSDataService,
+  params: SqlParametersList,
+  geoid: string,
+  table: string,
+): Promise<string | undefined> {
+  const executeParams: ExecuteStatementRequest = {
+    database: process.env.DATABASE_NAME ?? 'postgres',
+    resourceArn: process.env.RESOURCE_ARN ?? '',
+    schema: SCHEMA,
+    secretArn: process.env.CREDENTIALS_SECRET ?? '',
+    sql: `SELECT ${geoid}
+          FROM ${table}
+          WHERE geom && ST_SetSRID(ST_Point(:long, :lat), 4326)
+          LIMIT 1`,
```

Comment:
Will there ever be more than one shape that intersects a given point? If so, I'm concerned about throwing away that data by limiting this to the first result. If not, we should update the design doc, which says there can be multiple water systems returned for the /geolocate endpoint.

Reply:
Added a todo. Once we search by county, state that will be true (multiple water systems)